### DISCUSSION TITLE
Is there anyway to create a list which can allow a person access to my private repository?

### DISCUSSION THREAD

**Discussion by kleqing** [community/community#54035] [New to GitHub] [answered]
I and my friend use github for our work. We love how git and github work and we alway created many repository for our project. But some bad guys in my school want to steal it. So we have to changed it to private. But when we do that, me (or my friend) can not access to the project anymore. It is very hard for us to publish the code because it was private! Is there anyway for me to list which person can access to the private repository? Thank for advanced. I also sorry about my bad English too!

**Comment by airtower-luna** [comment, marked as answer]
You and your friend can [add each other as *collaborators*](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository) to your private repositories. Then you'll be able to see the repositories and work on them together.

**Reply by kleqing** [reply]
Thanks! Now I can publish the code easier with my friend without any issues!
